src/After.tsx

```diff
@@ -56,7 +56,7 @@ class Afterparty extends React.Component<any, any> {
     const { location } = this.props;
     const initialData = this.prefetcherCache[location.pathname] || data;
     return (
-      <Switch>
+      <Switch location={previousLocation || location}>
```

Comment:
So this will show the previous screen until the data is loaded? This isn't what we want. In the future, we'll use suspense to do this with a timeout.